### ISSUE TITLE
Broken build with incorrect Docusaurus link

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#3675]
It looks like https://github.com/Seneca-CDOT/telescope/pull/3674 and https://github.com/Seneca-CDOT/telescope/commit/9ec6b775406f1701315f6a48e7c8ea7fde348bae broke the build:

```
#24 119.5    -> linking to /docs/vi/docs/overview
#24 119.5 
#24 119.5     at throwError (/docs/node_modules/.pnpm/@docusaurus+logger@2.1.0/node_modules/@docusaurus/logger/lib/index.js:76:11)
#24 119.5     at handleBrokenLinks (/docs/node_modules/.pnpm/@docusaurus+core@2.1.0_tik67uczcphpud7o7cdxe2b2py/node_modules/@docusaurus/core/lib/server/brokenLinks.js:153:47)
#24 119.5     at async buildLocale (/docs/node_modules/.pnpm/@docusaurus+core@2.1.0_tik67uczcphpud7o7cdxe2b2py/node_modules/@docusaurus/core/lib/commands/build.js:177:5)
#24 119.5     at async tryToBuildLocale (/docs/node_modules/.pnpm/@docusaurus+core@2.1.0_tik67uczcphpud7o7cdxe2b2py/node_modules/@docusaurus/core/lib/commands/build.js:38:20)
#24 119.5     at async mapAsyncSequential (/docs/node_modules/.pnpm/@docusaurus+utils@2.1.0_@docusaurus+types@2.1.0/node_modules/@docusaurus/utils/lib/jsUtils.js:34:24)
#24 119.5     at async Command.build (/docs/node_modules/.pnpm/@docusaurus+core@2.1.0_tik67uczcphpud7o7cdxe2b2py/node_modules/@docusaurus/core/lib/commands/build.js:73:21)
```

I suspect that the leading `/` on the URL is what is causing it.  I'd try doing another change to use `to: 'docs/overview',` vs. `to: '/docs/overview',`, based on what I see in the `footer` here https://docusaurus.io/docs/api/docusaurus-config#themeConfig.

cc @cindyledev, @mnosov622, @Kevan-Y.

**Comment by mnosov622** [comment]
Oh, I'm so sorry, I didn't want to break the build :(

**Comment by humphd** [comment]
@mnosov622 don't worry, we all break the build at some point.  However, we can't leave it broken, since it blocks other devs.  This is a good chance for you to fix your second bug!  And we'll help.

**Comment by Kevan-Y** [comment]
Our docusaurus `baseUrl` is `/docs/` with the code added below
https://github.com/Seneca-CDOT/telescope/blob/9ec6b775406f1701315f6a48e7c8ea7fde348bae/src/web/docusaurus/docusaurus.config.js#L123

Now the route is `docs/docs/overview `which is a broken link.

We need to change it to ` to: '/overview'`, so when it build the route will be `/docs/overview`

**Comment by humphd** [comment]
Nice detective work, @Kevan-Y, thank you!  @mnosov622 I'll assign this to you, and you can try doing another fix.

Make sure you update your `master` branch before you start, so it picks up the current work we landed already.

Let us know if you need help.

**Comment by mnosov622** [comment]
Thank you ! 

I will do that.

**Comment by manekenpix** [comment]
Closed via #3678
